src/components/modal.tsx

```diff
@@ -3,7 +3,11 @@ import { useI18n } from '@wordpress/react-i18n';
 import { cx } from 'src/lib/cx';
 import type { ComponentProps } from 'react';
 
-export default function Modal( { className, ...rest }: ComponentProps< typeof WPModal > ) {
+export default function Modal( {
+	className,
+	isFullScreen,
```

Comment:
NIT Typically prefix `is` isn't used. For example `disabled`. And I think it makes sense, since this has better readability `<Modal fullScreen onRequestClose={ closeModal }>` instead of `<Modal isFullScreen onRequestClose={ closeModal }>`.

```suggestion
	fullScreen,
```